src/Microsoft.AspNetCore.Sockets.Client.Http/HttpConnection.cs

```diff
@@ -653,6 +496,20 @@ private ConnectionState ChangeState(ConnectionState to)
             }
         }
 
+        private void EnsureConnected()
+        {
+            if (_connectionState != ConnectionState.Connected)
+            {
+                throw new InvalidOperationException(
+                "Cannot send messages when the connection is not in the Connected state.");
+            }
+        }
+
+        void IDisposable.Dispose()
+        {
+            // We want to hide this since it noops, it's temporary until we remove IDisposable from IDuplexPipe
```

Comment:
File this somewhere so it doesn't get overlooked later

Reply:
https://github.com/dotnet/corefx/issues/27154